website/docs/developer-section/libraries/overview.md

```diff
@@ -1,17 +1,17 @@
 ---
-id: overview
+id: overview-lib
 title: Overview
 sidebar_label: Overview
 ---
 
-As a developer, you may find multiple repositories or libraries used throughout this documentation website and want to find out the different packages you can use to implement your own components. This section provides an overview of all the repositories and their respective documentation to help you get started.
+As a developer, you may find multiple repositories or libraries used throughout this documentation website and want to find out the different packages you can use to implement your own components. This section provides an overview of what you can build with all the repositories and their respective documentation for you to get started.
 
-Documentation to the repositories are in the following:
+You can achieve the following:
```

Comment:
Thanks for reviewing. Will it be okay to use this instead?

> Perform the tasks with the available repositories:

Because the bullet list below shows the tasks, followed by each repo: 

> - Create OpenAttestation (OA) documents such as transferable records or verifiable documents with the [OpenAttestation](http://localhost:3000/docs/developer-section/libraries/remote-files/open-attestation) core framework.
>
> - Create and verify OA documents as well as issue these documents to the blockchain in the command line interface using the [OpenAttestation (CLI)](http://localhost:3000/docs/developer-section/libraries/remote-files/open-attestation-cli).

Reply:
I see, in that case shall we go with "Using the repositories below, you can achieve the following:"?

Reply:
Yes, updated